packages/core/core/src/ParcelConfig.js

```diff
@@ -29,7 +29,7 @@ import ThrowableDiagnostic, {
 } from '@parcel/diagnostic';
 import json5 from 'json5';
 
-import {makeRe} from 'micromatch';
+import {globToRegex} from '@parcel/utils';
```

Comment:
Centralized all uses of micromatch into the `@parcel/utils` package so we only get one copy

Reply:
I recall that there was some breaking change in micromatch that thwarted a previous attempt to consolidate it (see [this comment](https://github.com/parcel-bundler/parcel/pull/6155#issuecomment-822626449) from @mischnic on a PR I forgot about 😨 ). Not sure if it's still relevant?

Reply:
At least one occurence was fixed in https://github.com/parcel-bundler/parcel/pull/6958